app/views/govuk_publishing_components/components/docs/cards.yml

```diff
@@ -68,6 +68,58 @@ examples:
             text: Housing and local services
             path: http://www.gov.uk
           description: Owning or renting and council services
+  one_column_layout_without_heading:
+    description: No border at the top of the list of cards is present when using a one column layout without a heading. 
+    data:
+      items:
+        - link:
+            text: Benefits
+            path: http://www.gov.uk
+          description: Includes eligibility, appeals, tax credits and Universal Credit
+        - link:
+            text: Births, deaths, marriages and&nbsp;care
+            path: http://www.gov.uk
+          description: Parenting, civil partnerships, divorce and Lasting Power of Attorney
+        - link:
+            text: Business and self-employed
+            path: http://www.gov.uk
+          description: Tools and guidance for businesses
+        - link:
+            text: Childcare and parenting
+            path: http://www.gov.uk
+          description: Includes giving birth, fostering, adopting, benefits for children, childcare and schools
+        - link:
+            text: Citizenship and living in the&nbsp;UK
+            path: http://www.gov.uk
+          description: Voting, community participation, life in the UK, international projects
+        - link:
+            text: Crime, justice and the&nbsp;law
+            path: http://www.gov.uk
+          description: Legal processes, courts and the police
+        - link:
+            text: Disabled people
+            path: http://www.gov.uk
+          description: Includes carers, your rights, benefits and the Equality Act
+        - link:
+            text: Driving and transport
+            path: http://www.gov.uk
+          description: Includes vehicle tax, MOT and driving licences
+        - link:
+            text: Education and learning
+            path: http://www.gov.uk
+          description: Includes student loans, admissions and apprenticeships
+        - link:
+            text: Employing people
+            path: http://www.gov.uk
+          description: Includes pay, contracts, hiring and redundancies
+        - link:
+            text: Environment and countryside
+            path: http://www.gov.uk
+          description: Includes flooding, recycling and wildlife
+        - link:
+            text: Housing and local services
+            path: http://www.gov.uk
+          description: Owning or renting and council services
```

Comment:
Would it be reasonable to trim the data in these examples down a bit? There's so many rows here the example doesn't fit onto my screen, making it harder to navigate this page.

Reply:
Yeah makes sense to me, I've made this change, looks much better in the component guide.